cmd/edgectl/aes_install.go

```diff
@@ -818,11 +741,17 @@ func (i *Installer) Quit() {
 	i.cancel()
 }
 
-func (i *Installer) ShowWrapped(text string) {
-	text = strings.Trim(text, "\n")                  // Drop leading and trailing newlines
-	for _, para := range strings.Split(text, "\n") { // Preserve newlines in the text
-		for _, line := range doWordWrap(para, "", 79) { // But wrap text too
-			i.show.Println(line)
+func (i *Installer) ShowWrapped(texts ...string) {
+	for _, text := range texts {
+		if text == "\n" {
+			i.show.Println()
```

Comment:
You don't need this special case. Passing the empty string yields a blank line, and I find that to be clearer at the call site. Non-empty arguments don't need a trailing newline, so passing in a lone newline looks weird.

Reply:
Ok, I'll fix this.